core/templates/dev/head/domain/collection/CollectionUpdateServiceSpec.js

```diff
@@ -55,7 +55,8 @@ describe('Collection update service', function() {
   it('should add/remove a new collection node to/from a collection',
       function() {
     expect(_sampleCollection.getExplorationIds()).toEqual(['exp_id0']);
-    CollectionUpdateService.addCollectionNode(_sampleCollection, 'exp_id1');
+    CollectionUpdateService.addCollectionNode(
+      _sampleCollection, 'exp_id1', undefined);
```

Comment:
Should the third arg be some reasonable non-null or non-undefined value? Otherwise this actually looks a bit like an error case.

Ditto below.